.nvmrc

```diff
@@ -1,2 +1,2 @@
-v16.5.0
+v18.15.0
```

Comment:
Was v20 a leg too far? It becomes the active LTS from 2023-10-24 already.

Reply:
cool